lib/PTEVal.ml

```diff
@@ -38,20 +45,25 @@ and pp_af ok = pp_int_field ok "af" (fun p -> p.af)
 and pp_db ok = pp_int_field ok "db" (fun p -> p.db)
 and pp_dbm ok = pp_int_field ok "dbm" (fun p -> p.dbm)
 and pp_el0 ok = pp_int_field ok "el0" (fun p -> p.el0)
+and pp_attrs ok = pp_field ok (fun a -> sprintf "attrs:(%s)" (String.concat ", " a))
+                    attrs_eq (fun p -> p.attrs)
 
 let set_oa p s = { p with oa = Misc.add_physical s; }
 
 let is_default t =
   let d = prot_default in
-  t.valid=d.valid && t.af=d.af && t.db=d.db && t.dbm=d.dbm && t.el0=d.el0
+  t.valid=d.valid && t.af=d.af && t.db=d.db && t.dbm=d.dbm && t.el0=d.el0 && t.attrs=[]
+
+let is_valid t = t.valid = 1
```

Comment:
If I am not wrong, this function `is_valid` is not called anywhere and is dead code.

Reply:
ack, this is now resolved.